### PR TITLE
Update clustered-mendix-runtime.md

### PR DESCRIPTION
**Pull request by NielsKunst**
Removed outdated information. All cluster management activities are performed by any of the runtime instances.

See https://mendix.slack.com/archives/C3T260XGA/p1731415485630109

### CODE REVIEW

**Comment by MarkvanMents** [comment]
Check if cluster leader is mentioned anywhere else.

File: content/en/docs/refguide/runtime/_index.md [hunk 1]
```diff
@@ -31,14 +31,13 @@ The user interacts with the Mendix Client, which then makes requests, via the [r
 
 Passing state from the Runtime Server to the Mendix Client enables the Runtime Server to be stateless, which means that any Runtime Server instance can respond to a request from the Mendix Client. A load balancer decides which Runtime Server instance will respond to a request. When a user session ends, the Runtime Server removes references to that session.
 
-If there is more than one instance of an app, one of the instances is the *Cluster Leader*. The Runtime Server in that instance is responsible for a number of activities which cannot easily be distributed. These include:
+If there is more than one instance of an app, one of the instances is the *Cluster Leader*. This is responsible for Performing database synchronization tasks. It also schedules a number of activities which can then be picked up by any node. These include:
```
Comment (hugomendix) [inline content/en/docs/refguide/runtime/_index.md]: Consider rephrasing "a number of activities" to "a number of cluster management activities".

File: content/en/docs/refguide/runtime/custom-settings/_index.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ The following custom settings can be configured:
...
 | <a id="SessionKeepAliveUpdatesInterval" href="#SessionKeepAliveUpdatesInterval">SessionKeepAliveUpdatesInterval</a> | Defines how often a runtime writes session LastActive dates in its memory back to the database. | one sixth of the value configured for the `SessionTimeout` setting; if the `SessionTimeout` is not set, this value defaults to 100000 (100 seconds) |
-| <a id="SessionTimeout" href="#SessionTimeout">SessionTimeout</a> | Defines after how much time session becomes invalid (in milliseconds). After that timeout a session becomes applicable for removal. The session will not be destroyed until the next time the cluster manager evaluates the active sessions. | 600000 (10 minutes) |
+| <a id="SessionTimeout" href="#SessionTimeout">SessionTimeout</a> | Defines after how much time session becomes invalid (in milliseconds). After that timeout a session becomes applicable for removal. The session will not be destroyed until the next time the a scheduled task runs to evaluates the active sessions. | 600000 (10 minutes) |
```
Comment (hugomendix) [inline content/en/docs/refguide/runtime/custom-settings/_index.md]: Typos:
- "the a" -> "the"
- "evaluates" -> "evaluate" (or maybe even better to use "clean up")

Comment (hugomendix) [inline content/en/docs/refguide/runtime/custom-settings/_index.md]: Would it be better to use "eligible" (for removal) instead of "applicable"?

Comment (MarkvanMents) [reply content/en/docs/refguide/runtime/custom-settings/_index.md]: Yes, I'll do that. Thanks for the thorough proofread!

Comment (MarkvanMents) [reply content/en/docs/refguide/runtime/custom-settings/_index.md]: Thanks for catching that.

File: content/en/docs/refguide/runtime/_index.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/refguide/runtime/_index.md]: Yes, I'll do that.

**Review by MarkvanMents — APPROVED** [review]
Thanks for the thorough review. I've updated the doc in this commit: 7827dbac1f478b6077cc63cad5ccfff268014ecf
OK to publish?

**Pull request merged**